extension/content/personas.css

```diff
@@ -90,6 +90,20 @@
   font-weight: bold !important;
 }
 
+#personas-toolbar-button.seamonkey, #personas-toolbar-button.seamonkey:hover, #personas-toolbar-button.seamonkey:hover:active {
+  list-style-image: url("chrome://personas/content/personas_30x30.png");
+  -moz-image-region: rect(0, 30px, 30px, 0px);  
+}
+
+toolbar[iconsize="small"] #personas-toolbar-button.seamonkey, toolbar[iconsize="small"] #personas-toolbar-button.seamonkey:hover, toolbar[iconsize="small"] #personas-toolbar-button.seamonkey:hover:active {
+  list-style-image: url("chrome://personas/content/personas_20x20.png");
+}
+
+#personas-menu.seamonkey {
```

Comment:
Not sure what this has to do with `hover` or `active`. Those selectors are not used here.